buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -242,6 +243,28 @@ public void beforeExecute(Task task) {}
         );
     }
 
+    private void stopCluster(ElasticsearchCluster each, boolean taskFailed) {
+        if (allowClusterToSurvive) {
+            logger.info("Not stopping clusters, disabled by property");
+            if (taskFailed) {
+                // task failed or this is the last one to stop
+                for (int i=1 ; ; i += i) {
```

Comment:
I'm not sure I see the point of this increasing backoff loop. If we are going to just let the build hang here there's no need to display this message more than once.

Reply:
I had it like that initially, but combined with the interactive status line I found it to lack visibility.